Lib/urllib/parse.py

```diff
@@ -39,12 +40,35 @@
            "parse_qsl", "quote", "quote_plus", "quote_from_bytes",
            "unquote", "unquote_plus", "unquote_to_bytes",
            "DefragResult", "ParseResult", "SplitResult",
+           "SchemeFlag", "RELATIVE", "NETLOC", "PARAMS", "UNIVERSAL",
```

Comment:
Lets not pollute `__all__` with the CONSTANT_NAMES. People shouldn't really use `from urllib.parse import *` but if they do they shouldn't get these, just SchemeFlag.

Reply:
@ethanfurman thoughts? I remember you suggested that these should be exported as such for code like

```python
urlparse(uri_string, flags=UNIVERSAL)
```

or similar. I'm fine either way, but do agree that the namespace would be cleaner were the flags not exported individually.

Reply:
Putting them in `globals()` is not for the `from ... import *` case, since, as @gpshead said, folks should not be doing that; putting them in `globals()` is to enable `urlparse.RELATIVE` usage, much like we have `re.IGNORECASE` and not `re.RegexFlag.IGNORECASE`.